src/chrome/content/rules/Gdl.de.xml

```diff
@@ -0,0 +1,160 @@
+<!--
+	Other GDL rulesets:
+		- Gdl-hof.de.xml
+		- Gdl-kempten.de.xml
+		- Gdl-og-gera.de.xml 
+		- Gdl-sbahn-berlin.de.xml
+
+	Valid, but redirects to plain HTTP:
+		- gdl-og-mannheim.de
+
+	Server-side SSL Error:
+		- gdl-aachen.de
+		- gdl-bayern.de
+		- gdl-friedrichshafen.de
+		- gdl.koeln (geoTLD)
+		- gdl-mühldorf.de (xn- -gdl-mhldorf-xhb.de)
+		- gdl-nn.de
+		- gdl-plochingen.de
+		- gdl-tuebingen.de
+
+	Mismatched:
+		- umfrage.gdl.de
+		- gdl-bremen.de
+		- gdl-garmisch.de
+		- gdl-heilbronn.de
+		- gdl-kassel.de
+		- gdl-kornwestheim.de
+		- gdl-m.de
+		- gdl-regensburg.de
+		- gdl-ulm.de
+		- gdl-weiden-schwandorf.de
+		- gdl-wuerzburg.de
+
+	Certificate Error Unknown Issuer:
+		- sentry.gdl.de
+		
+	Plaintext response on HTTPS:
```

Comment:
What does this mean?

Reply:
@Bisaloo It effectively means a plaintext HTTP server is listening on port 443 due to misconfiguration.